doc/README.md

```diff
@@ -213,6 +213,10 @@ the graph, controlled by the *nodecount* option.
 
 * **"(inline)" Edge Marker**: the call has been inlined into the caller.
 
+* **Green Nodes/Edges**: when [comparing two profiles](#comparing-profiles), the
```

Comment:
I think a larger change should be considered (basically, everywhere "red" is mentioned, "green" should also be mentioned). To minimize the number of top-level bullets, it may also make sense to replace the "Thicker & Redder Edges" and "Thinner & Greyer Edges" bullets with a "Edge color and weight bullet".

Also note that edge weight and font size both indicate a large absolute value.

Reply:
Should I have a separate bullet point for "comparing two profiles"? It will be difficult to describe those properties combining both the normal situation and the "comparison mode". For example, a red edge means larger value in either situation, but "more resources were used along that path" only applies to the normal situation and does not apply to the "comparison mode".

Reply:
Comparison is one way in which negative values can appear in the profiles which we typically work with. But pprof is a general tool; there's no reason that profiles with negative values cannot be viewed with the pprof CLI, so green can show up when one is not using comparison mode if the input profile contains negative values.

Reply:
Done.